src/components/autocomplete/js/autocompleteController.js

```diff
@@ -392,9 +408,13 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
         event.stopPropagation();
         event.preventDefault();
         clearValue();
-        ctrl.matches = [];
-        ctrl.hidden  = true;
-        ctrl.index   = getDefaultIndex();
+
+        // Force the component to blur if they hit escape
+        doBlur(true);
+
+        //ctrl.matches = [];
+        //ctrl.hidden  = true;
+        //ctrl.index   = getDefaultIndex();
```

Comment:
Please remove commented code.  If we need it, we can pull it up in git, but we typically don't want to keep commented code in our documents.